pkg/deployments/action-ids/polygon/expected-collisions.json

```diff
@@ -0,0 +1,240 @@
+    },
+    {
+      "taskId": "20220318-batch-relayer-v2",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "unwrapWstETH(address,address,uint256,uint256)",
+      "useAdaptor": true
+    }
+  ],
+  "0xcab56cf0a06b55508156a5c957a970efad1443bad7856dbe80ec11a370f80a89": [
+    {
+      "taskId": "20211203-batch-relayer",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "vaultPermit(address,address,uint256,uint256,uint8,bytes32,bytes32)",
+      "useAdaptor": true
+    },
+    {
+      "taskId": "20220318-batch-relayer-v2",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "vaultPermit(address,address,uint256,uint256,uint8,bytes32,bytes32)",
+      "useAdaptor": true
+    }
+  ],
+  "0xedebced60d7da90f933bbff86bec5241792e17792f152ae92fe1114b921c5907": [
+    {
+      "taskId": "20211203-batch-relayer",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "vaultPermitDAI(address,address,uint256,uint256,bool,uint8,bytes32,bytes32)",
+      "useAdaptor": true
+    },
+    {
+      "taskId": "20220318-batch-relayer-v2",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "vaultPermitDAI(address,address,uint256,uint256,bool,uint8,bytes32,bytes32)",
+      "useAdaptor": true
+    }
+  ],
+  "0x2d09a22fbe9c7d158e7987034d52ddbb9cc91614360aebee2b7672dfb8df50be": [
+    {
+      "taskId": "20211203-batch-relayer",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "wrapAaveDynamicToken(address,address,address,uint256,bool,uint256)",
+      "useAdaptor": true
+    },
+    {
+      "taskId": "20220318-batch-relayer-v2",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "wrapAaveDynamicToken(address,address,address,uint256,bool,uint256)",
+      "useAdaptor": true
+    }
+  ],
+  "0x312291dc2ab61f228f04827afccdb688df88526f6e687fb057f1a4ea94814e63": [
+    {
+      "taskId": "20211203-batch-relayer",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "wrapStETH(address,address,uint256,uint256)",
+      "useAdaptor": true
+    },
+    {
+      "taskId": "20220318-batch-relayer-v2",
+      "contractName": "BatchRelayerLibrary",
+      "signature": "wrapStETH(address,address,uint256,uint256)",
+      "useAdaptor": true
+    }
+  ],
+  "0xc0821b38b909d4450a0639d4a7e4922284925325006a23c1ae2fa1e729c9b781": [
+    {
+      "taskId": "20211203-batch-relayer",
+      "contractName": "BalancerRelayer",
+      "signature": "multicall(bytes[])",
+      "useAdaptor": true
+    },
+    {
+      "taskId": "20220318-batch-relayer-v2",
+      "contractName": "BalancerRelayer",
+      "signature": "multicall(bytes[])",
+      "useAdaptor": true
+    }
+  ],
+  "0x527e5a8917fef9f19f55ea7d3f9dc98c00e4836cf918d2af144d2ef91076a53c": [
+    {
+      "taskId": "20211208-aave-linear-pool",
+      "contractName": "AaveLinearPoolFactory",
+      "signature": "create(string,string,address,address,uint256,uint256,address)",
+      "useAdaptor": true
```

Comment:
That's true but I don't think there's any good way to filter out the non-authenticated functions and there's not much point in having two rounds of filtering where one will do.

Reply:
It might be more "truthy" to change this to `isAuthorizerAware` and flip these values. It avoids a false positive (as these factories really aren't aware of the authorizer) but we have the same issue of adaptor + unauthenticated contracts being mixed.

Reply:
Nah, this is fine I think